webpack.config.js

```diff
@@ -19,9 +19,7 @@ const config = {
         library: 'manifold',
         umdNamedDefine: true
     },
-    node: {
-        net: 'empty'
-    },
+    mode: 'development',

```

Comment:
Do you think it might be better to default to production? I'm not entirely sure what the differences are, but since we use this build process to create the package for distribution, I'm thinking a production-oriented build might be more appropriate than a dev-oriented build.

Reply:
(And maybe we need a different action to do a dev build for development purposes... I'm sure there must be a best practice around this, but I'm not sure what it is -- maybe we need to look for precedents elsewhere).

Reply:
Definitely, I'm not sure why I put development here! Fixed.